src/app/Dashboard/Dashboard.tsx

```diff
@@ -41,7 +41,7 @@ import { TargetView } from '@app/TargetView/TargetView';
 export interface DashboardProps {}
 
 export const Dashboard: React.FunctionComponent<DashboardProps> = (props) => {
-  return <TargetView pageTitle="Dashboard" compactSelect={true} />;
+  return <TargetView pageTitle="Dashboard" compactSelect={false} />;
```

Comment:
Some reasons to expand the targetSelect out here :D ??

Reply:
It was a bit redundant, but in the last set of commits I've removed the empty state from the dashboard itself and put an empty state into the `TargetSelect`. The expansion button is now always visible (unless in `simple` style) and the empty state is shown if nothing is selected. This cleans up the code a bit too, there's no more `props.simple || selected === NO_TARGET ? undefined : whatever` everywhere.

Reply:
Nicee very much cleaner :D